lib/src/models/transaction_uploader.dart

```diff
@@ -131,18 +132,23 @@ class TransactionUploader {
     //  throw StateError('Unable to validate chunk: $_chunkIndex');
 
     // Catch network errors and turn them into objects with status -1 and an error message.
-    final res = await _api.post('chunk', body: json.encode(chunk));
-
-    _lastRequestTimeEnd = DateTime.now().millisecondsSinceEpoch;
-    lastResponseStatus = res.statusCode;
-
-    if (lastResponseStatus == 200) {
-      _chunkIndex++;
-    } else {
-      lastResponseError = getResponseError(res);
-      if (FATAL_CHUNK_UPLOAD_ERRORS.contains(lastResponseError)) {
-        throw StateError(
-            'Fatal error uploading chunk: $_chunkIndex: $lastResponseError');
+    Response? res;
+    try {
+      res = await _api.post('chunk', body: json.encode(chunk));
+    } catch (e) {
+      print("Error posting to endpoint, " + e.toString());
```

Comment:
`Error posting to endpoint, ` -> `Error posting to /chunk endpoint: `

Reply:
[Added](https://github.com/ardriveapp/arweave-dart/pull/14/commits/84418832ad14fc5d1d9d564d96022f7b0957fb20)